src/Cache.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex;
+
+use Composer\Cache as BaseCache;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class Cache extends BaseCache
+{
+    private static $lowestTags = [
+        'symfony/symfony' => 'v3.4.0',
```

Comment:
what about all the components ?

Reply:
symfony/symfony is found in all components, thanks to the "replace" statement I suppose.

Reply:
sure, but this will remove the outdated tags from the main package. I don't think it will remove outdated tags of the component packages. Meaning you only remove half of what you could be removing here

Reply:
It does for sure. Actually removing all versions of symfony/symfony (since it's not used) frees very little memory. The memory+CPU are reduced a lot *because* lowest tags for components are dropped.